docs/highlights/2020-09-18-adaptive-concurrency.md

```diff
@@ -0,0 +1,67 @@
+---
+last_modified_on: "2020-09-18"
+$schema: ".schema.json"
+title: "Adaptive concurrency. Optimizing performance & reliability."
+description: "How Vector optimizes performance and increases reliability of your entire observability infrastructure."
+author_github: "https://github.com/binarylogic"
+pr_numbers: [3094]
+release: "0.11.0"
+hide_on_release_notes: false
+tags: ["type: announcement", "domain: networking"]
+---
+
+Vector 0.11 includes a new adaptive conccurrency feature designed to
+automatically optimize network bandwidth, backoff when it senses trouble,
+and make your observability infrastructure significantly more performant and
+reliable.
+
+## Adaptive Concurrency. The Elevator Pitch.
+
+We cover this feature in-depth in our [announcement blog post][announcement],
+but here's the skinny.
+
+One of the most difficult things to do in distributed systems is coordinating
+communication in a way that optimizes performance without risking reliability.
+This is often achieved through rate-limits. In theory, rate-limits sound simple,
+but in practice they're often the largest source of performance degredation.
+
+For example, setting rate-limits too high risks overwhelming services causing
+them fail. And setting them too low artifically limits performance.
```

Comment:
them to fail.

Reply:
I'd combine this into a single sentence:

"Setting rate-limits too high, for example, risks overwhelming services, causing them to fail, while setting them too low artificially limits performance."

Also note the spelling fix on "artificially."